ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -45,7 +45,7 @@
     <input data-sly-test="${!useTextarea}"
            title="${text.tooltipVisible ? '' : text.tooltipText}"
            class="cmp-adaptiveform-textinput__widget"
-           aria-label="${text.label.visible && text.label.value}"
+           aria-label="${text.label.value}"  
```

Comment:
is aria-label still there when label is hidden?

Reply:
Yes by default it is always taking it true